wp-resources/plugins/sil-dictionary-webonary/webonary/Webonary_Cloud.php

```diff
@@ -284,40 +284,36 @@ public static function getLanguageName(string $code, string $default = ''): stri
 	}
 
 	/**
-	 * @param string|null $dictionaryId
+	 * @param string $dictionaryId
 	 * @return ICloudDictionary|stdClass|null
 	 */
-	public static function getDictionary(?string $dictionaryId = null): ICloudDictionary|stdClass|null
+	public static function getDictionaryById(string $dictionaryId): ICloudDictionary|stdClass|null
 	{
-		if (!is_null(self::$dictionary))
-			return self::$dictionary;
-
-		$dictionary = get_option('dictionary');
-		if (!empty($dictionary)) {
-			self::$dictionary = $dictionary;
+		$request = self::$doGetDictionary . '/' . $dictionaryId;
+		$response = self::remoteGetJson($request);
+		if (self::isValidDictionary($response)) {
+			update_option('dictionary', $response);
+			return $response;
 		}
-		else {
+		
+		return null;
+	}
 
-			if (empty($dictionaryId))
-				$dictionaryId = self::getBlogDictionaryId();
+	/**
+	 * @return ICloudDictionary|stdClass|null
+	 */
+	public static function getDictionary(): ICloudDictionary|stdClass|null
+	{
+			if (!is_null(self::$dictionary))
+				return self::$dictionary;
 
-			$request = self::$doGetDictionary . '/' . $dictionaryId;
-			$response = self::remoteGetJson($request);
-			if (self::isValidDictionary($response)) {
-				update_option('dictionary', $response);
-				self::$dictionary = $response;
+			$dictionary = get_option('dictionary');
+			if (empty($dictionary)) {
+				$dictionary = self::getDictionaryById(self::getBlogDictionaryId());
 			}
-		}
-
-		return self::$dictionary;
-	}
 
-	public static function getTotalCount($doAction, $apiParams = array()): int
```

Comment:
The method `getTotalCount` is removed by this PR, but it is still used in `Webonary_Cloud.php` and `browseview_func.php`

Reply:
Oh my! That was by mistake. Fixed. Thank you @phillip-hopper